From 1b7d0124f63623fbc4db291f87f855c909b0bde9 Mon Sep 17 00:00:00 2001
From: Steve deRosier <Steve.Derosier@lairdtech.com>
Date: Mon, 13 Jul 2015 17:45:24 -0700
Subject: [PATCH] laird: wb45n: dt: Disable unused mmc1 to prevent spurious
 probes

By disabling the mmc1 device in the devicetree, we keep it from trying to
probe the interface. This is generally only noticed when MMC_DEBUG is turned
on, but also happens to result in a 1.5 Mbps speed up on the wifi chip.

Bug: 7265
---
 arch/arm/boot/dts/at91sam9g25ek.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/at91sam9g25ek.dts b/arch/arm/boot/dts/at91sam9g25ek.dts
index c5ab16f..a846ff2 100644
--- a/arch/arm/boot/dts/at91sam9g25ek.dts
+++ b/arch/arm/boot/dts/at91sam9g25ek.dts
@@ -13,4 +13,11 @@
 / {
 	model = "Atmel AT91SAM9G25-EK";
 	compatible = "atmel,at91sam9g25ek", "atmel,at91sam9x5ek", "atmel,at91sam9x5", "atmel,at91sam9";
+	ahb {
+		apb {
+			mmc1: mmc@f000c000 {
+				status = "disabled";
+			};
+		};
+	};
 };
-- 
1.8.3.2

